sdk/nodejs/lambda/lambdaMixins.ts

```diff
@@ -0,0 +1,365 @@
+// Copyright 2016-2018, Pulumi Corporation.
```

Comment:
Not for this commit - but the more we add these overlays - the more I feel like we should just move "overlays" into the `sdk` folder next to the generated code instead of generating them from themselves.  That will make authoring more sane, and avoid this strange duplication of files.  Now that we check in generated files, there are few downsides to doing this.

Reply:
YES.  I 100% agree.  it's super annoying to do things like we do today :)